tests/phpunit/tests/link/getPreviousCommentsLink.php

```diff
@@ -28,11 +28,29 @@ public function test_page_should_default_to_1_when_no_cpage_query_var_is_found()
 		$cpage = get_query_var( 'cpage' );
 		set_query_var( 'cpage', '' );
 
-		$link = get_previous_comments_link( 'Next' );
+		$link = get_previous_comments_link( 'Previous' );
 
 		// Technically, it returns null here.
 		$this->assertNull( $link );
 
 		set_query_var( 'cpage', $cpage );
 	}
+
+	/**
+	 * @ticket 60806
+	 */
+	public function test_page_should_respect_value_of_page_argument() {
+		$p = self::factory()->post->create();
+		$this->go_to( get_permalink( $p ) );
+
+		// Check setting the query var is ignored.
+		$cpage = get_query_var( 'cpage' );
```

Comment:
The following will make it easier to follow. I think the assertion needs to run last in the case it fails so the cleanup still happens as planned:
```suggestion
		$old_cpage = get_query_var( 'cpage' );
```

Reply:
It makes sense to me. I changed that in this commit: https://github.com/WordPress/wordpress-develop/pull/7275/commits/656630aeda0d5a6ddcba75097803b18ce3c833c5. I also took the opportunity to improve the old tests in this other one: https://github.com/WordPress/wordpress-develop/pull/7275/commits/bea742841653c63cf93295bb61dc13dc38f3b213. I can revert them if this is not the appropriate place.

Reply:
Sure, I should mention the same issue existed in other places. Good call fixing all occurrences.